sass/px-dropdown-content-sketch.scss

```diff
@@ -52,6 +52,20 @@ Predix branding rules go in px-dropdown-predix.scss, not in this file.
   margin-right: 5px;
 }
 
+
+input[type="search"] {
+  margin-left: 8px;
```

Comment:
Does the way I did it is ok, or you meant something else, like add class directly to the html element?